re.rizin.cutter.plugin.rz-ghidra.yml

```diff
@@ -0,0 +1,31 @@
+id: re.rizin.cutter.plugin.rz-ghidra
+branch: stable
+runtime: re.rizin.cutter
+runtime-version: stable
+appstream-compose: false
+sdk: org.kde.Sdk//5.15-21.08
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.llvm12
+build-options:
+  prepend-path: /usr/lib/sdk/llvm12/bin
+  prepend-ld-library-path: /usr/lib/sdk/llvm12/lib
+
+build-extension: true
+modules:
+  - name: rz-ghidra
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DBUILD_CUTTER_PLUGIN=ON
+      - -DCMAKE_INSTALL_PREFIX=/app/extensions
```

Comment:
1. this is not the proper prefix. 
2. prefix should be set in the top level `build-options` and should be  `/app/extensions/rz-ghidra`
3. actually this is duplicated and inconsitent with line 23.

Reply:
Thanks for the tips. Is there a documentation about all thoses practices ? I did not find any :/